src/main/kotlin/lotto/LottoGame.kt

```diff
@@ -0,0 +1,26 @@
+package lotto
+
+import WinningResult
+import lotto.LottoConstants.LOTTO_PRICE
+
+class LottoGame {
+    fun purchase(paidMoney: Int): List<Lotto> {
+        val purchaseTime = paidMoney / LOTTO_PRICE
+        require(paidMoney >= LOTTO_PRICE) { "로또 구입 금액은 1000원 이상이어야 합니다." }
```

Comment:
연산을 하기전에 먼저 유효성을 검증해주면 불필요한 연산을 할 필요가 없을 것 같아요. 😃

```suggestion
        require(paidMoney >= LOTTO_PRICE) { "로또 구입 금액은 1000원 이상이어야 합니다." }
        val purchaseTime = paidMoney / LOTTO_PRICE
```